system-tests/projects/session-and-origin-e2e-specs/cypress/e2e/session/errors.cy.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Used in cy-in-cy tests in @packages/app.
+ */
+it('setup has failing command', () => {
+  cy.session('session_1', () => {
+    cy.get('does_not_exist', { timeout: 500 })
```

Comment:
```suggestion
    cy.get('does_not_exist', { timeout: 1 })
```

Any reason not to have it fail faster?

Reply:
good idea